eppo_core/src/event_ingestion/event_delivery.rs

```diff
@@ -7,20 +10,48 @@ use std::collections::HashSet;
 use url::Url;
 use uuid::Uuid;
 
+const MAX_EVENT_SERIALIZED_LENGTH: usize = 4096;
```

Comment:
Could you explain how this constant was chosen? Is it a limit imposed by the server or just picked arbitrarily? (would be great to have it in a comment)

Reply:
yup, this is a limit imposed by the backend. its a number we thought was reasonably large enough to hold most events. we don't expect people to start submitting really large event payloads (that's not the intention), and if they do, then we maybe need to help/educate them on how to use it